app/GoogleReCaptchaWrapper.tsx

```diff
@@ -0,0 +1,28 @@
+"use client"
+
+import { GoogleReCaptchaProvider } from "@google-recaptcha/react"
+
+export default function GoogleReCaptchaWrapper({ children }: { children: React.ReactNode }) {
+  // This key is used by the frontend to load and execute the reCAPTCHA script.
+  const siteKey = process.env.NEXT_PUBLIC_RECAPTCHA_SITE_KEY!
```

Comment:
In TypeScript, forcing a value to be non-null (!) can sometimes crash the app in production.
It's safer to provide a fallback empty string.
```suggestion
  const siteKey = process.env.NEXT_PUBLIC_RECAPTCHA_SITE_KEY || "";
```

Reply:
Thanks for the suggestion! Updated the code to use a safe fallback (|| "") instead of the non-null assertion to prevent runtime issues